INSTALL

```diff
@@ -19,7 +19,7 @@ Optional dependencies:
   - FreeType (for the TTF fonts plugin)
   - ICONV (for the TTF fonts plugin and only needed when using a non unicode compatible TLK)
   - (lib)VLC 2.0+ (for movies in some mac distributions of the games)
-  - OpenGL drivers and GLEW
+  - OpenGL/OpenGLES drivers
 
```

Comment:
Please also include your note somewhere, so other builders will have an easier time:
> The build system prefers the Broadcom GLES libraries when a RPI system is detected, but this can be overriden by adding -DDISABLE_VIDEOCORE=1 - and is necessary for the Pi4.

Reply:
Yes, that would be useful - I'll include some build instructions.